frontend/screens/LoginPage.js

```diff
@@ -1,136 +1,183 @@
 import * as React from "react";
 import { Image } from "expo-image";
-import { StyleSheet, View, Text, TouchableOpacity, ScrollView } from "react-native";
+import { StyleSheet, View, Text, TouchableOpacity, ScrollView, TextInput, Alert } from "react-native";
 import { useNavigation } from "@react-navigation/native";
 
-import Depth1Frame6 from "../components/Depth1Frame6";
 import { Color, Padding, FontSize, Border, FontFamily } from "../GlobalStyles";
 
 const LoginPage = () => {
   const navigation = useNavigation();
 
+  const [username, setUsername] = React.useState("");
+  const [password, setPassword] = React.useState("");
+
+  //It handle the login functionality
+  const handleInfluencerLogin = async () => {
+    if (username.trim() != "" && password.trim() != "") {
+      try {
+        const response = await fetch(`${process.env.API_ENDPOINT}/influencers/login`, {
```

Comment:
Lets define a `controller` folder under the `src` and create `loginController.js` file under it and move all these login api calls in there, it will help us reuse the code and avoid the duplication